src/features/HomePage/index.tsx

```diff
@@ -17,16 +16,13 @@ import PageWithTransition from '@/components/PageWithTransition';
 import Welcome from './components/Welcome';
 
 const HomePage = () => {
-  const hasUnreadMessages = useAppSelector(selectHasUnreadMessages);
-  const userRole = useAppSelector(selectUserRole);
-
+  const unreadMessagesFound = useAppSelector(selectHasUnreadMessages);
```

Comment:
Again I always cry about naming. :D
But i would want to prefix boolean variable-names always with
is/has/should => hasUnreadMessages

Reply:
No problem! 😃 That's a good convention, this one seems to have slipped.